### PR TITLE
Add all properties to GraphQLError

### PR DESCRIPTION
**Pull request by ducan-ne**
This allow reading property `code`, `extensions` from `GraphQLError`

### CODE REVIEW

File: src/Draqula.ts [hunk 1]
```diff
@@ -141,7 +141,12 @@ export default class Draqula {
 				// GraphQL servers don't fail the request, even if resolvers are failing
 				// They still return 200 status code, but add `errors` field in the response
 				if (Array.isArray(response.errors)) {
-					const errors = response.errors.map((error: any) => new Error(error.message));
+					const errors = response.errors.map((error: any) => {
+						let err = new Error(error.message) as any;
```
Comment (sindresorhus) [inline src/Draqula.ts]:
```suggestion
						let error2 = new Error(error.message) as any;
```

File: src/Draqula.ts [hunk 2]
```diff
@@ -141,7 +141,13 @@ export default class Draqula {
 				// GraphQL servers don't fail the request, even if resolvers are failing
 				// They still return 200 status code, but add `errors` field in the response
 				if (Array.isArray(response.errors)) {
-					const errors = response.errors.map((error: any) => new Error(error.message));
+					const errors = response.errors.map((error: any) => {
+						let error2 = new Error(error.message) as any;
```
Comment (vadimdemedes) [inline src/Draqula.ts]: `let` => `const`, since you don't reassign `error2`.

File: src/Draqula.ts [hunk 3]
```diff
@@ -141,7 +141,13 @@ export default class Draqula {
...
+						for (let key in error) {
+							error2[key] = error[key];
+						}
+						return error2
```
Comment (vadimdemedes) [inline src/Draqula.ts:151]: Could you add a newline above?

File: test/useQuery.js [hunk 4]
```diff
@@ -942,10 +942,12 @@ test('handle GQL errors', async t => {
 		.reply(200, {
 			errors: [
 				{
-					message: 'Error 1'
+					message: 'Error 1',
+					extensions: {code: 'ERROR_1'}
```
Comment (vadimdemedes) [inline test/useQuery.js:946]: Which tool is returning `extensions` field? I don't think I've seen it before.

Comment (stefanvanburen) [reply test/useQuery.js:946]: It's at least in the latest draft version of the spec: https://graphql.github.io/graphql-spec/June2018/#example-fce18; graphql-go optionally adds this to errors: https://github.com/graph-gophers/graphql-go/commit/62c5401acea6852dbfda63863fe8bf13ab5a00cf

Comment (ducan-ne) [reply test/useQuery.js:946]: it come from apollo

**Comment by vadimdemedes** [comment]
Thanks @quaj!

**Comment by vadimdemedes** [comment]
Will be out in `0.2.1`.

**Pull request merged**